script/main.py

```diff
@@ -243,17 +244,37 @@ def average_age(conn, gender):
     print('Average age of women: {value:.2f} years'.format(value = average_age_of_women))
   else:
     print('Overall average age: {value:.2f} years'.format(value = average_age_overall))
+
+
+def select_all_sorted_cities(conn):
+  cur = conn.cursor()
+  cur.execute("SELECT location_city, count(location_city) FROM users GROUP BY location_city ORDER BY count(location_city) DESC")
```

Comment:
Use the `LIMIT` option in `SELECT` command

Reply:
what scope to give?

Reply:
You get this number as a parameter

Reply:
aaaaaa :D ok :D